exporter/awskinesisexporter/factory.go

```diff
@@ -74,6 +74,7 @@ func NewTracesExporter(ctx context.Context, params component.ExporterCreateSetti
 		params,
 		conf,
 		exp.ConsumeTraces,
+		exporterhelper.WithStart(exp.Start),
```

Comment:
I noticed that the start method wasn't being used, are you okay for me to include the fix in this PR @Aneurysm9 ?

Reply:
Does that impact the ability to use the region config being added here?  It doesn't look like it to me, so I'd think to make this a separate change, but I could be missing something.

Reply:
That makes sense with me, I just saw it and wanted to fix it up.

Let me move it to a seperate PR.